dom0 tests FAIL: `test_sd_vm_apt_sources` (`apt-test` instead of expected `apt.freedom.press`)

## Description

CI failed recently, I believe on main as well ([see the log](https://ws-ci-runner.securedrop.org/2025-05-30-063003879727-22933834c7592f0bb0966ce404232ec0ad1880f0-Qubes_4.2_D-update_20250530043423.log.txt)).

The failure is as follows:

`AssertionError: 'Description: https://apt.freedom.press bookworm/main amd64 Packages\n' not found in <'apt-get indextargets' output>`

<details><summary>Full CI failure</summary>

```
INFO:[2025-05-30-07:31:09:305975] ======================================================================
INFO:[2025-05-30-07:31:09:306010] FAIL: test_sd_vm_apt_sources (test_vms_platform.SD_VM_Platform_Tests.test_sd_vm_apt_sources)
INFO:[2025-05-30-07:31:09:306050] Test that the three templates we install our apt sources into are correct
INFO:[2025-05-30-07:31:09:306083] ----------------------------------------------------------------------
INFO:[2025-05-30-07:31:09:306139] Traceback (most recent call last):
INFO:[2025-05-30-07:31:09:306180]   File "/home/user/securedrop-workstation/tests/test_vms_platform.py", line 197, in test_sd_vm_apt_sources
INFO:[2025-05-30-07:31:09:306217]     self.assertIn(
INFO:[2025-05-30-07:31:09:306253] AssertionError: 'Description: https://apt.freedom.press bookworm/main amd64 Packages\n' not found in 'MetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: https://apt-test.freedom.press bookworm/main amd64 Packages\nURI: https://apt-test.freedom.press/dists/bookworm/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/apt-test.freedom.press_dists_bookworm_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nOrigin: SecureDrop\nTrusted: yes\nArchitecture: amd64\nBase-URI: https://apt-test.freedom.press/dists/bookworm/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: https://apt-test.freedom.press/\nSite: https://apt-test.freedom.press\nSourcesentry: /etc/apt/sources.list.d/apt-test_freedom_press.sources:1\nTarget-Of: deb\n\nMetaKey: nightlies/binary-amd64/Packages\nShortDesc: Packages\nDescription: https://apt-test.freedom.press bookworm/nightlies amd64 Packages\nURI: https://apt-test.freedom.press/dists/bookworm/nightlies/binary-amd64/Packages\nFilename: /var/lib/apt/lists/apt-test.freedom.press_dists_bookworm_nightlies_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nOrigin: SecureDrop\nTrusted: yes\nArchitecture: amd64\nBase-URI: https://apt-test.freedom.press/dists/bookworm/\nBy-Hash: yes\nComponent: nightlies\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: https://apt-test.freedom.press/\nSite: https://apt-test.freedom.press\nSourcesentry: /etc/apt/sources.list.d/apt-test_freedom_press.sources:1\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm/main amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Debian\nOrigin: Debian\nSuite: stable\nTrusted: yes\nVersion: 12.11\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:24\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm/contrib amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Debian\nOrigin: Debian\nSuite: stable\nTrusted: yes\nVersion: 12.11\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:24\nTarget-Of: deb\n\nMetaKey: non-free/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm/non-free amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm/non-free/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm_non-free_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Debian\nOrigin: Debian\nSuite: stable\nTrusted: yes\nVersion: 12.11\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm/\nBy-Hash: yes\nComponent: non-free\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:24\nTarget-Of: deb\n\nMetaKey: non-free-firmware/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm/non-free-firmware amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm/non-free-firmware/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm_non-free-firmware_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Debian\nOrigin: Debian\nSuite: stable\nTrusted: yes\nVersion: 12.11\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm/\nBy-Hash: yes\nComponent: non-free-firmware\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:24\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm-updates/main amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm-updates/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm-updates_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-updates\nLabel: Debian\nOrigin: Debian\nSuite: stable-updates\nTrusted: yes\nVersion: 12-updates\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm-updates/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-updates\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:25\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian-security bookworm-security/main amd64 Packages\nURI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian-security_dists_bookworm-security_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-security\nLabel: Debian-Security\nOrigin: Debian\nSuite: stable-security\nTrusted: yes\nVersion: 12\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-security\nRepo-URI: tor+https://deb.debian.org/debian-security/\nSite: tor+https://deb.debian.org/debian-security\nSourcesentry: /etc/apt/sources.list.d/debian.list:26\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian-security bookworm-security/contrib amd64 Packages\nURI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian-security_dists_bookworm-security_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-security\nLabel: Debian-Security\nOrigin: Debian\nSuite: stable-security\nTrusted: yes\nVersion: 12\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-security\nRepo-URI: tor+https://deb.debian.org/debian-security/\nSite: tor+https://deb.debian.org/debian-security\nSourcesentry: /etc/apt/sources.list.d/debian.list:26\nTarget-Of: deb\n\nMetaKey: non-free-firmware/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian-security bookworm-security/non-free-firmware amd64 Packages\nURI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/non-free-firmware/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian-security_dists_bookworm-security_non-free-firmware_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-security\nLabel: Debian-Security\nOrigin: Debian\nSuite: stable-security\nTrusted: yes\nVersion: 12\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian-security/dists/bookworm-security/\nBy-Hash: yes\nComponent: non-free-firmware\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-security\nRepo-URI: tor+https://deb.debian.org/debian-security/\nSite: tor+https://deb.debian.org/debian-security\nSourcesentry: /etc/apt/sources.list.d/debian.list:26\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm-backports/main amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm-backports/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm-backports_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports\nLabel: Debian Backports\nOrigin: Debian Backports\nSuite: stable-backports\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm-backports/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:27\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm-backports/contrib amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm-backports/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm-backports_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports\nLabel: Debian Backports\nOrigin: Debian Backports\nSuite: stable-backports\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm-backports/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:27\nTarget-Of: deb\n\nMetaKey: non-free/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm-backports/non-free amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm-backports/non-free/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm-backports_non-free_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports\nLabel: Debian Backports\nOrigin: Debian Backports\nSuite: stable-backports\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm-backports/\nBy-Hash: yes\nComponent: non-free\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:27\nTarget-Of: deb\n\nMetaKey: non-free-firmware/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.debian.org/debian bookworm-backports/non-free-firmware amd64 Packages\nURI: tor+https://deb.debian.org/debian/dists/bookworm-backports/non-free-firmware/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.debian.org_debian_dists_bookworm-backports_non-free-firmware_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports\nLabel: Debian Backports\nOrigin: Debian Backports\nSuite: stable-backports\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.debian.org/debian/dists/bookworm-backports/\nBy-Hash: yes\nComponent: non-free-firmware\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports\nRepo-URI: tor+https://deb.debian.org/debian/\nSite: tor+https://deb.debian.org/debian\nSourcesentry: /etc/apt/sources.list.d/debian.list:27\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://fasttrack.debian.net/debian-fasttrack bookworm-fasttrack/main amd64 Packages\nURI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/fasttrack.debian.net_debian-fasttrack_dists_bookworm-fasttrack_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-fasttrack\nLabel: Fasttrack\nOrigin: Debian\nSuite: bookworm-fasttrack\nTrusted: yes\nVersion: 12.0\nArchitecture: amd64\nBase-URI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-fasttrack\nRepo-URI: tor+https://fasttrack.debian.net/debian-fasttrack/\nSite: tor+https://fasttrack.debian.net/debian-fasttrack\nSourcesentry: /etc/apt/sources.list.d/debian.list:28\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://fasttrack.debian.net/debian-fasttrack bookworm-fasttrack/contrib amd64 Packages\nURI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/fasttrack.debian.net_debian-fasttrack_dists_bookworm-fasttrack_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-fasttrack\nLabel: Fasttrack\nOrigin: Debian\nSuite: bookworm-fasttrack\nTrusted: yes\nVersion: 12.0\nArchitecture: amd64\nBase-URI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-fasttrack\nRepo-URI: tor+https://fasttrack.debian.net/debian-fasttrack/\nSite: tor+https://fasttrack.debian.net/debian-fasttrack\nSourcesentry: /etc/apt/sources.list.d/debian.list:28\nTarget-Of: deb\n\nMetaKey: non-free/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://fasttrack.debian.net/debian-fasttrack bookworm-fasttrack/non-free amd64 Packages\nURI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/non-free/binary-amd64/Packages\nFilename: /var/lib/apt/lists/fasttrack.debian.net_debian-fasttrack_dists_bookworm-fasttrack_non-free_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-fasttrack\nLabel: Fasttrack\nOrigin: Debian\nSuite: bookworm-fasttrack\nTrusted: yes\nVersion: 12.0\nArchitecture: amd64\nBase-URI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-fasttrack/\nBy-Hash: yes\nComponent: non-free\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-fasttrack\nRepo-URI: tor+https://fasttrack.debian.net/debian-fasttrack/\nSite: tor+https://fasttrack.debian.net/debian-fasttrack\nSourcesentry: /etc/apt/sources.list.d/debian.list:28\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://fasttrack.debian.net/debian-fasttrack bookworm-backports-staging/main amd64 Packages\nURI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-backports-staging/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/fasttrack.debian.net_debian-fasttrack_dists_bookworm-backports-staging_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports-staging\nLabel: BackportsStaging\nOrigin: Debian\nSuite: bookworm-backports-staging\nTrusted: yes\nVersion: 12.0\nArchitecture: amd64\nBase-URI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-backports-staging/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports-staging\nRepo-URI: tor+https://fasttrack.debian.net/debian-fasttrack/\nSite: tor+https://fasttrack.debian.net/debian-fasttrack\nSourcesentry: /etc/apt/sources.list.d/debian.list:29\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://fasttrack.debian.net/debian-fasttrack bookworm-backports-staging/contrib amd64 Packages\nURI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-backports-staging/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/fasttrack.debian.net_debian-fasttrack_dists_bookworm-backports-staging_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm-backports-staging\nLabel: BackportsStaging\nOrigin: Debian\nSuite: bookworm-backports-staging\nTrusted: yes\nVersion: 12.0\nArchitecture: amd64\nBase-URI: tor+https://fasttrack.debian.net/debian-fasttrack/dists/bookworm-backports-staging/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm-backports-staging\nRepo-URI: tor+https://fasttrack.debian.net/debian-fasttrack/\nSite: tor+https://fasttrack.debian.net/debian-fasttrack\nSourcesentry: /etc/apt/sources.list.d/debian.list:29\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.kicksecure.com bookworm/main amd64 Packages\nURI: tor+https://deb.kicksecure.com/dists/bookworm/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.kicksecure.com_dists_bookworm_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Whonix\nOrigin: whonix\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.kicksecure.com/dists/bookworm/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.kicksecure.com/\nSite: tor+https://deb.kicksecure.com\nSourcesentry: /etc/apt/sources.list.d/derivative.list:17\nTarget-Of: deb\n\nMetaKey: contrib/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.kicksecure.com bookworm/contrib amd64 Packages\nURI: tor+https://deb.kicksecure.com/dists/bookworm/contrib/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.kicksecure.com_dists_bookworm_contrib_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Whonix\nOrigin: whonix\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.kicksecure.com/dists/bookworm/\nBy-Hash: yes\nComponent: contrib\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.kicksecure.com/\nSite: tor+https://deb.kicksecure.com\nSourcesentry: /etc/apt/sources.list.d/derivative.list:17\nTarget-Of: deb\n\nMetaKey: non-free/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.kicksecure.com bookworm/non-free amd64 Packages\nURI: tor+https://deb.kicksecure.com/dists/bookworm/non-free/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.kicksecure.com_dists_bookworm_non-free_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Whonix\nOrigin: whonix\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.kicksecure.com/dists/bookworm/\nBy-Hash: yes\nComponent: non-free\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.kicksecure.com/\nSite: tor+https://deb.kicksecure.com\nSourcesentry: /etc/apt/sources.list.d/derivative.list:17\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: tor+https://deb.whonix.org bookworm/main amd64 Packages\nURI: tor+https://deb.whonix.org/dists/bookworm/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.whonix.org_dists_bookworm_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Whonix\nOrigin: whonix\nTrusted: yes\nArchitecture: amd64\nBase-URI: tor+https://deb.whonix.org/dists/bookworm/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: tor+https://deb.whonix.org/\nSite: tor+https://deb.whonix.org\nSourcesentry: /etc/apt/sources.list.d/derivative.list:20\nTarget-Of: deb\n\nMetaKey: main/binary-amd64/Packages\nShortDesc: Packages\nDescription: https://deb.qubes-os.org/r4.2/vm bookworm/main amd64 Packages\nURI: https://deb.qubes-os.org/r4.2/vm/dists/bookworm/main/binary-amd64/Packages\nFilename: /var/lib/apt/lists/deb.qubes-os.org_r4.2_vm_dists_bookworm_main_binary-amd64_Packages\nOptional: no\nKeepCompressed: no\nCodename: bookworm\nLabel: Qubes Debian\nOrigin: Qubes OS Debian\nTrusted: yes\nArchitecture: amd64\nBase-URI: https://deb.qubes-os.org/r4.2/vm/dists/bookworm/\nBy-Hash: yes\nComponent: main\nCompressionTypes: xz bz2 lzma gz lz4 zst uncompressed\nCreated-By: Packages\nDefaultEnabled: yes\nFallback-Of: \nIdentifier: Packages\nKeepCompressedAs: lz4 zst gz xz bz2 lzma uncompressed\nPDiffs: yes\nRelease: bookworm\nRepo-URI: https://deb.qubes-os.org/r4.2/vm/\nSite: https://deb.qubes-os.org/r4.2/vm\nSourcesentry: /etc/apt/sources.list.d/qubes-r4.list:2\nTarget-Of: deb'
INFO:[2025-05-30-07:31:09:306532] 
```

</details>

## Steps to Reproduce

Please specify your environment if that is necessary to reproduce the bug (if in doubt, include it).

## Expected Behavior


## Actual Behavior

Please provide screenshots where appropriate.

## Comments

Suggestions to fix, any other relevant information.

--- a/dom0/runner.py
+++ b/dom0/runner.py
@@ -197,6 +197,10 @@ class QubesCI:
         self.run_cmd("make clone")
         self.run_cmd("make dev")
         self.shutdown_sd_vms()
+
+        # Simulate updater. Workaround for https://github.com/freedomofpress/securedrop-workstation/issues/1333
+        self.run_cmd("sudo qubes-vm-update --show-output --targets whonix-gateway-17 --force-update")
+
         self.run_cmd("make test", env={"CI": "true"})
 
     def systemInfo(self):

